camel/agents/task_agent.py

```diff
@@ -67,6 +70,9 @@ def __init__(
             role_type=RoleType.ASSISTANT,
             content="You can make a task more specific.",
         )
+        if output_language != "English":
+            system_message.content += f"And you should output task specification in {output_language}"
+
```

Comment:
Let's define a function named `reset_output_language` in the `ChatAgent` class to achieve this. Would be great if we can you a pytest function to test it as well.